contributors.md

```diff
@@ -104,7 +104,7 @@ The following people are not part of the development team, but have been contrib
 * Umar Ahmed (umar-ahmed) - MacOS file watcher
 * Andrew Arnold (fidwell) - Added window support for more scenery groups.
 * Josh Trzebiatowski (trzejos) - Ride and scenery filtering
-* (kyphii) - Extended color selection, reversed ride vehicles, misc.
+* (kyphii) - Extended color selection
```

Comment:
Looks like an accidental revert.

```suggestion
* (kyphii) - Extended color selection, reversed ride vehicles, misc.
```

Reply:
Yeah there is a lot of reverts in this pr.